lib/timeline/component/item/Item.js

```diff
@@ -145,16 +145,26 @@ Item.prototype.repositionY = function() {
  */
 Item.prototype._repaintDragCenter = function () {
   if (this.selected && this.options.editable.updateTime && !this.dom.dragCenter) {
-    var me = this;
-
+    var me = this
     // create and show drag area
     var dragCenter = document.createElement('div');
     dragCenter.className = 'vis-drag-center';
     dragCenter.dragCenterItem = this;
+    this.hammer = new Hammer(dragCenter)
 
-    new Hammer(dragCenter).on('doubletap', function (event) {
+    this.hammer.on('tap', function (event) {
+      me.parent.itemSet.body.emitter.emit('click',  {
+        event: event,
+        item: me.id
+      });
+    });
+    this.hammer.on('doubletap', function (event) {
       event.stopPropagation();
       me.parent.itemSet._onUpdateItem(me);
+      me.parent.itemSet.body.emitter.emit('doubleClick', {
```

Comment:
I am trusting that this block is correct; it is congruent to `click` above, so it looks OK.